### PR TITLE
fix: add missing styles from MP to Aquarium

### PR DESCRIPTION
**Pull request by gabyzif**
This PR adds the styles that are in style.less to Aquarium

Link for the PR in nancy: https://git.corp.mparticle.com/mParticle/mPServer/pull/27691 
Closes https://go.mparticle.com/work/UNI-219

Attached some screenshots of the style changes vs prod

Screenshot 2024-04-08 at 5 59 33 PM
Screenshot 2024-04-08 at 5 59 39 PM
Screenshot 2024-04-08 at 5 59 57 PM

### CODE REVIEW

File: src/components/navigation/GlobalNavigation/global-navigation.css [hunk 1]
```diff
@@ -147,6 +147,12 @@
   height: var(--size-lg) !important;
 }
 
+.globalNavigation__searchIcon {
+  width: 22px !important;
+  height: 22px !important;
+  margin-left: -3px;
```
Comment (jared-dickman) [inline src/components/navigation/GlobalNavigation/global-navigation.css:153]: full disclosure negative margin always makes me cringe. its not possible to add spacing to the parent element?

**Comment by jared-dickman** [comment]
yes should def be released as a feature branch and tested locally in each platform
for completeness, a link to the pr env/testing bucket in each env would be amazing

File: src/components/navigation/GlobalNavigation/global-navigation.css (hunk 1 above)
Comment (jared-dickman) [reply src/components/navigation/GlobalNavigation/global-navigation.css:153]: infact it looks off center to me now, is this desired?

with change
image


from [chromatic](https://www.chromatic.com/component?appId=65aec861fa4adbf0708aebd0&csfId=aquarium-navigation-globalnavigation--mp&buildNumber=520&k=66104a709b6e2d44c92102e2-1200px-interactive-true&h=7&b=-4)/main
image

File: src/components/navigation/GlobalNavigation/global-navigation.css [hunk 2]
```diff
@@ -249,15 +255,19 @@
 }
 
 .globalNavigation__iconLabel {
+  font-family: var(--font-family) ;
```
Comment (jared-dickman) [inline src/components/navigation/GlobalNavigation/global-navigation.css:258]: was this not being applied as the default?

File: src/components/navigation/GlobalNavigation/global-navigation.css [hunk 3]
```diff
@@ -292,13 +302,17 @@
...
 }
 
 .globalNavigation__divider {
   border-bottom: 1px solid var(--color-border-secondary);
   margin-right: var(--margin-sm);
   margin-left: var(--margin-sm);
-  margin-bottom: var(--margin-md);
+  /* margin-bottom: var(--margin-md); */
+  margin-bottom: 0;
+  margin-top: -3px;
```
Comment (jared-dickman) [inline src/components/navigation/GlobalNavigation/global-navigation.css:315]: can the same effect be achieved by reducing the padding-bottom on globalNavigation__suiteLogo?

File: src/components/navigation/GlobalNavigation/global-navigation.css [hunk 4]
```diff
@@ -292,13 +302,17 @@
...
+
 }
 
 .globalNavigation__divider {
   border-bottom: 1px solid var(--color-border-secondary);
   margin-right: var(--margin-sm);
   margin-left: var(--margin-sm);
-  margin-bottom: var(--margin-md);
+  /* margin-bottom: var(--margin-md); */
+  margin-bottom: 0;
```
Comment (jared-dickman) [inline src/components/navigation/GlobalNavigation/global-navigation.css:314]: turning this on and off in dev tools doesnt seem to impact anything, is this controlling something specifically?

File: src/components/navigation/GlobalNavigation/global-navigation.css (hunk 1 above)
Comment (tibuurcio) [reply src/components/navigation/GlobalNavigation/global-navigation.css:153]: It gives me pain as well 😅

Comment (gabyzif) [reply src/components/navigation/GlobalNavigation/global-navigation.css:153]: 100%, but I think we can polish this part in a few weeks/week bc we'll be working on this for the POC https://mparticle-eng.atlassian.net/browse/UNI-301?atlOrigin=eyJpIjoiMzg4Yzk4YTdmZWJkNDU3NmJlNmNjYjNlYjdlODgwODIiLCJwIjoiamlyYS1zbGFjay1pbnQifQ

**Review by tibuurcio — APPROVED** [review]

File: src/components/navigation/GlobalNavigation/global-navigation.css (hunk 1 above)
Comment (jared-dickman) [reply src/components/navigation/GlobalNavigation/global-navigation.css:153]: this is a nit, but the spacing doesnt seem better to me now, should def have design sign off on this before merging it in

one thing i've learned is to look for the spacing north/east/south/west to be consistent. admittedly whats in production now doesnt look perfect, but the spacing now wrt the divider isnt event, its much closer to the button than the suite logo

image

**Pull request closed without merging**